corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/menu.less

```diff
@@ -15,7 +15,12 @@
   #module-case-list-map {
     height: calc(~"100vh - 65px");
     position: sticky !important; // Has to be important so the leaflet code does not set it to relative
-    top: 50px; // based on the height of the crumbs bar
+    top: 47.125px; // based on the height of the crumbs bar
+    z-index: 1;
+
+    @media (max-width: 767px) {
```

Comment:
nit: I assume the 767 is from bootstrap's "small devices" width definition? I think bootstrap 5 has a way of defining it directly via `breakpoint`but it's not available for bootstrap3 https://getbootstrap.com/docs/5.0/layout/breakpoints/

What do you think about defining this as a constant instead?

Reply:
Yep, that's the size (and what was decided with AEs was likely the right size). I can actually use the predefined bootstrap constant, it turns out https://github.com/dimagi/commcare-hq/pull/33351/commits/8cef043e8fff84b7d0c6b81691b50a86c5968f0b